cypress/support/commands.js

```diff
@@ -32,10 +32,29 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+Cypress.Commands.add('register', (email, username, password) => {
```

Comment:
Use object shorthand for properties. Instead of `email: email, username: username, password: password` you can just write `email, username, password`.